app/jobs/modsulator_job.rb

```diff
@@ -121,6 +113,32 @@ def update_metadata(druid, xml_string, log)
     end
   end
 
+  # Open a new version for the given object if it is in the accessioned state.
+  def version_object(dor_object, original_filename, user_login, log)
```

Comment:
I tend to agree with avoiding unused arguments.  It doesn't appear that this method requires a `user_login`.  Is it possible to remove that from the method signature?